Unify cellular example (#210)
ARMmbed/mbed-os-example-cellular: README.md

## Getting started

This particular cellular application uses a cellular network and network-socket APIs that are part of [`mbed-os`](https://github.com/ARMmbed/mbed-os).

The program uses a [cellular modem driver](https://github.com/ARMmbed/mbed-os/tree/master/connectivity/cellular/include/cellular/framework/API)
using an external IP stack (LWIP) standard 3GPP AT 27.007 AT commands to setup the cellular modem and registers to the network.

After registration, the driver opens a point-to-point protocol (PPP) pipe using LWIP with the cellular modem and connects
to internet. This driver currently supports UART data connection type only between your cellular modem and MCU.

For more information on Arm Mbed OS cellular APIs and porting guide, please visit the
[Mbed OS cellular API](https://os.mbed.com/docs/mbed-os/latest/apis/cellular-networking.html) and
[Mbed OS cellular porting guide](https://os.mbed.com/docs/mbed-os/latest/porting/cellular-device-porting.html).

### Board support

The cellular modem driver in this example uses PPP with an Mbed-supported external LwIP stack.

> It supports targets when modem exists on the Mbed Enabled target as opposed to plug-in modules (shields).

Reviewer: ```suggestion
The driver supports Mbed Enabled targets when a modem exists on the device. Plug-in modules (shields) are not supported.
```

It wasn't clear to me whether it supported shields or not from the way this sentence was written. My suggestion assumes we don't support shields.

        }
```

## Running the example

When example application is running information about activity is printed over the serial connection.

**Note:** The default serial baudrate has been set to 115200.

Please have a client open and connected to the board. You may use:

- [Tera Term](https://ttssh2.osdn.jp/index.html.en) for windows

- screen or minicom for Linux (example usage: `screen /dev/serial/<your board> 115200`)

> - mbed tools have terminal command `mbed term -b 115200`

Reviewer: ```suggestion
- mbed tools has a terminal command `mbed term -b 115200`
```
